### ISSUE TITLE
Hi Rachel!!

### ISSUE THREAD

**Issue by ghost** [rachsmithcodes/ama#7]
Hi Rachel,

I'm an aussie! I studied engineering but have been unable to get a gig in my area (aerospace) but have been doing more and more front end work. It can be frustrating to try and put myself out there (and a little bit intimidating). What has been your best return on investment in terms of building you reputation as a great front-end dev?

Thanks

**Comment by rachsmithcodes** [comment]
heya @jmzwar 
Earlier in my career my main focus was on building my reputation in the effort to being more employable. The most beneficial step to achieving this at that stage was keeping a well up to date and well presented portfolio. I also changed jobs fairly frequently early in my career (via freelancing and contract work) to try and build up lots of experience at different ad agencies. 
Later on, I wanted to start _contributing more_ publicly to the developer community, so I started publishing work on <a href="http://codepen.io>CodePen</a>. My motivation behind doing this wasn't really about gaining exposure as a developer but it turns out this was actually the most beneficial to me in building a reputation. My work on CodePen led to job opportunities, writing and speaking opportunities - <a href="http://codepen.io/rachsmith/post/last-year-i-joined-codepen-what-happened-next-will-blow-you-away">I wrote about it here</a>. So I guess my first suggestion would be to find somewhere to publish your work, **and just start doing it**. I know it can be intimidating, but I've had 0 regrets about putting my work online, especially in a place like CodePen where the community is so supportive. You might prefer to publish things you've made on github or your own website, but the point is to just your work out there.

My second suggestion on how to build your reputation as a developer is to be social, be nice to people, and try to to make meaningful connections with people. My most recent job and the new job I am about to start I was offered not only because of my work, but because I knew and was already friendly with the people who hired me. After moving to the USA and speaking at and going to conferences here I made a lot of really great friends and connections that have led to other opportunities. Maybe meeting people in real life is not as appealing to you (I get that, I'm actually a very shy person and I have to work on pushing through the uncomfortableness of meeting new people) but I think being social online can also really help. That has been my experience anyway :)
